api-inference-community/manage.py

```diff
@@ -63,6 +65,30 @@ def show(args):
                         print(" " * 4, key.value)
 
 
+def resolve(model_id: str) -> [str, str]:
+    info = HfApi().model_info(model_id)
+    task = info.pipeline_tag
+    framework = None
+    frameworks = {
+        "flair",
+        "speechbrain",
+        "allennlp",
+        "asteroid",
+        "espnet",
+        "sentence-transformers",
+        "spacy",
+    }
+    for tag in info.tags:
+        if tag in frameworks:
+            if framework is None:
+                framework = tag
+            else:
+                raise Exception(
+                    "This model seems to implement 2 frameworks, we cannot infer"
```

Comment:
Just a comment: I wonder if this can become an issue later on with models such as `sentence-transformers` that work fine in both transformers and sentence-transformers. 

`modelInfo` currently does not return the library, but I think it should. It's already there, we just need to add it to the returned modelInfo. I can send a PR for that next week.

Reply:
@julien-c is implementing `library_name` or something similar, so we should be able to switch to that and avoid this edge case.

Reply:
This is already done now :)

Reply:
You can do `info.library_name`